### DISCUSSION TITLE
MAPS: e1m1 tweaks

### DISCUSSION THREAD

**Discussion by RandomBrushes** [MissLav/LibreQuake#186] [General]
I've been working on some visual tweaks to e1m1. This includes:

- lighting tweaks, introducing some color touches
- minor tweaks to brushes to, e.g., avoid z-fighting
- reworking the outdoor rock walls for an organic look

Gameplay is unchanged.

I developed a habit of creating "cloud backups" for stuff that isn't yet in in git to prevent computer mishaps and enable feedback, so why not persist this here in a thread?

Current "cloud backup":
[e1m1-tweaks-wip.zip](https://github.com/user-attachments/files/17356189/e1m1-tweaks-wip.zip)

Some images:
![spasm0000](https://github.com/user-attachments/assets/736ff1a9-049c-4a09-a767-05a4f740e75d)
![spasm0001](https://github.com/user-attachments/assets/d0cf7d4e-4b72-4124-a3c8-c18860186244)
![spasm0002](https://github.com/user-attachments/assets/dd4c5907-a29d-4ef9-ac7b-e683562d5195)
![spasm0003](https://github.com/user-attachments/assets/6c488510-f907-4fef-9708-2353874d6f3b)
![spasm0004](https://github.com/user-attachments/assets/fc5dad6a-cd76-44ec-b85b-7224ca6d095c)
![spasm0005](https://github.com/user-attachments/assets/a18e81cc-ef3d-449c-b97b-f7deeb4af8a1)

**Comment by RandomBrushes** [comment]
I noticed that currently, one can simply run through the prison cell area without visiting the second floor (missing some items and perhaps a secret). Perhaps this prison area is a neat environment to introduce the key game mechanic?

![to-key-or-not-to-key](https://github.com/user-attachments/assets/1b927174-72b2-4221-a9fe-4663c5cb6cfa)

**Reply by Nolcoz** [reply]
Thats a really good idea. Seems really good for introducing the key mecanic. 
Maybe if you make it so the player sees the silver door first and then when they turn back the see the lift, it could work. Otherwise its possible they just grab the key first and use it on the door without realising it was needed

**Reply by RandomBrushes** [reply]
It turns out that e1m2 is designed to introduce keys, so e1m1 shouldn't hasten the introduction. However, it might still make sense to place some desirable item up there to motivate players to explore the second floor.

**Comment by RandomBrushes** [comment]
Handing over to ZungryWare to implement further rework ideas, latest sources enclosed.

[e1m1-tweaks-wip.zip](https://github.com/user-attachments/files/17361661/e1m1-tweaks-wip.zip)

**Comment by lavenderdotpet** [comment]
https://github.com/lavenderdotpet/LibreQuake/pull/190 resolved here
